Lib/fontParts/test/test_world.py

```diff
@@ -292,8 +292,13 @@ def tearDown(self):
         import shutil
         shutil.rmtree(self.font_dir)
 
-    def test_font_open(self):
+    def test_font_open_path(self):
         OpenFont(self.font_path)
+        
+    def test_font_open_object(self):
+        from fontTools.ufoLib import UFOReader
+        obj = UFOReader(self.font_path)
```

Comment:
not a UFOReader but a defcon font object, only in fontShell, in other implantation it would be the implementation font object.